### ISSUE TITLE
How to run non embeded app

### ISSUE THREAD

**Issue by anurag3386** [Shopify/shopify-app-template-php#419]
I am trying to setup non embedded app with the help shopify cli3 using php template.
I have disabled embedded app from the shopify section but app still not working as non embedded. Please help what change we need to do at script level under the laravel.

**Comment by shahzadwaris** [comment]
@anurag3386  Did you find out any solution for this? I'm also facing the same issue.

**Comment by anurag3386** [comment]
Hello @shahzadwaris 

I already fixed the same after lots reading the steps in code and do the same thing.

Thanks
Anurag

**Comment by shahzadwaris** [comment]
@ellowwwteam did you add the `HOST` with your custom domain in the `.env` file?

**Comment by ellowwwteam** [comment]
@shahzadwaris Yes, I have update the host in .env file. Any other changes need fixing the non embedded app?

**Comment by khanakia** [comment]
App is continuously refreshing and redirecting again and again after i made it Non Embeddable

**Comment by shahzadwaris** [comment]
Did you follow the above instructions?

**Comment by sk29110** [comment]
I started working on the none embeded app. Call to a member function send() on null. @shahzadwaris you are very confident about this can work as none embeded app. How do you do the last step? 

An extra step would be changing the redirect URL in /api/auth/callback route to your customer route.
This is done.

**Comment by shinyatentree** [comment]
Any thought? @shahzadwaris

**Comment by kamalsharma9392** [comment]
Hey @sk29110 & @shahzadwaris , did you guys able to solve this issue, I am also facing the same issue

**Comment by sushansht** [comment]
I did the following steps and it worked: 
1. From partner in app setup change embedded to false

2. In AppServiceProvider.php, set host to url of the site like 'https://example .com", set custom domain to ["example.com"], set is embeded to false and run config cache. You can set these value from env too.

3. In appbrideprovider.js change force redirect to false.

Then it should work as expected.

**Comment by veed76** [comment]
any luck ?

**Comment by novikor** [comment]
True.
Replacing all the `Utils::getEmbeddedAppUrl` redirects with redirect to route that corresponds to your app logic, e.g. `route('home')`, and setting `isEmbeddedApp: false` in the AppServiceProvider `Context::initialize` call.

There are just several places to edit in https://github.com/Shopify/shopify-app-template-php/blob/dabab615b1135cc4fa831e8496decb6a4b379a04/web/routes/web.php#L77

In general, this template is almost useless for a non-embedded app. Its only value is for copying and modifying the Oauth part.

> Okay, I found the solution.
> 
>     1. Log in to your Partner account and select your app.
> 
>     2. Click on App Setup from the left sidebar and scroll to the bottom.
> 
> 
>     3. Embedded app: Click on manage and turn it off.
> 
>     4. Modify the code in AppServiceProvider.php
>        `Context::initialize( env('SHOPIFY_API_KEY', 'not_defined'), env('SHOPIFY_API_SECRET', 'not_defined'), env('SCOPES', 'not_defined'), $host, new DbSessionStorage(), ApiVersion::LATEST, false, false, null, '', null, (array)$customDomain, );`
>        An extra step would be changing the redirect URL in /api/auth/callback route to your customer route.
>        This is done.
